### DISCUSSION TITLE
Alternative bank with extra exercises?

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/linear-algebra#301] [General]
See #294.

I've already slipped in CheckIt "outcomes" for a generated midterm/final exam, and there are various good ideas for alternative/supplemental exercises that don't exactly align with our outcomes.

What do we think about maintaining two banks? The first bank is the canonical bank of exercises aligned with our outcomes, and the second bank is for the extra stuff that instructors might find useful. For backwards design purposes, I think I like having a very clear picture of how we expect to measure student learning for our outcomes via a CheckIt bank, but there's uses for other types of generated exercises/assessments besides those as well.

**Comment by siwelwerd** [comment]
How do we want to package this?  I don't think I'd want this linked in to the activity book just to keep things clean and tidy.  However, an extra button on the [landing page](https://teambasedinquirylearning.github.io/linear-algebra/) for "Instructor Features", with a bank of "Supplemental Exercises" might work?

**Reply by StevenClontz** [reply]
I was thinking about a link in this existing drop-down, though if we have other supplemental resources we could add another dropdown:

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/7ca7f7f3-7cf2-42a5-9b10-b0e23ed6aee7)
